.cursor/commit-rules.mdc

```diff
@@ -0,0 +1,82 @@
+---
+description: Commit rules for cursor
+alwaysApply: true
```

Comment:
This is a lot of tokens to always add to the context. Perhaps we should make this apply only when generating commit messages?

Reply:
Updated!